web/src/server_events_dispatch.js

```diff
@@ -743,6 +743,9 @@ export function dispatch_normal_event(event) {
                 break;
             }
 
+            // TODO: Move privacy_settings_schema and PrivacySettings here from
+            // settings_account when this file is converted to typescript, and
+            // use them instead of `privacy_settings`
```

Comment:
We have a loose convention for TypeScript todo comments: `// TODO/typescript: …`.